README.md

```diff
@@ -12,7 +12,7 @@ The goal of the project is to develop a simplified version of the Instagram appl
 3. Leave comments to a posted picture
 
 ## Additional features
-1. Cannot repost a picture without crediting the original post
+1. The ability to search other users
 
```

Comment:
you can also add the other feature that we added other than the core features

You can see on the slides that the core features are:
* Post Picture
* Follow User
* Comment

Check the website and check what other feature we've implemented or you can look through the api's documentation